terraform-infra-approvals/ccpay-bulkscanning-app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "azurerm_template_deployment"}
```

Comment:
I can’t find this in your repo?
can you link me to it please?

Reply:
It's here:
https://github.com/hmcts/ccpay-bulkscanning-app/blob/demo/infrastructure/main.tf#L144

not really keen on adding this for a new repo =/
as far as I can tell it's all supported by terraform

If you could write a terraform module that does what the arm template is doing, then we could add that to the global whitelist, here are most of the resources you would need, reach out if you need any review or if it doesn't work:
https://www.terraform.io/docs/providers/azurerm/r/api_management_product.html
https://www.terraform.io/docs/providers/azurerm/r/api_management_group.html
https://www.terraform.io/docs/providers/azurerm/r/api_management_api.html
https://www.terraform.io/docs/providers/azurerm/r/api_management_api_operation.html
https://www.terraform.io/docs/providers/azurerm/r/api_management_api_policy.html

Reply:
HI Tim,
I am more than happy to do this in later time but right now two of our services are having failing builds and we are currently very close to integration phase with some third parties. We create a tech debt ticket and work on this in later sprints. I would appreciate if you could white-list these two services for now.